src/docs/DynamicInfoStores.docs.mdx

```diff
@@ -0,0 +1,53 @@
+import { Meta, Canvas } from "@storybook/blocks";
+import * as stories from "./stories/DynamicInfoStores.stories.svelte";
+
+<Meta title="Examples/Dynamic Info With Stores" />
+
+# Dynamic Info With Stores
+
+Here is a basic writable/derived Svelte store setup for displaying dynamic information, based on source data.
+
+## src/data/data.json
+
+```json
+[
+  {
+    "label": "Bar Chart",
+    "value": "barChart",
+    "metric": 100
+  },
+  {
+    "label": "Column Chart",
+    "value": "columnChart",
+    "metric": 200
+  },
+  {
+    "label": "Heatmap",
+    "value": "heatmap",
+    "metric": 300
+  }
+]
+```
+
+## src/lib/stores/selected.js
+
+```js
+import { writable, derived } from "svelte/store";
+import data from "$data/data.json";
+
+// use key to access object (instead of .find())
+data.reduce((map, d) => {
+  map[d.value] = d;
+  return map;
+}, data);
```

Comment:
only one tiny note here: I think it might be just a bit cleaner to follow a more functional programming paradigm and avoid mutating the original array and instead create a new object like this:
```js
const dataLookup = data.reduce((map, d) => {
  map[d.value] = d;
  return map;
}, {});
```